app/views/assessments/new.html.erb

```diff
@@ -0,0 +1,108 @@
+    <header class="panel-header">
+      <h2 class="panel-header__title">Profile</h2>
+    </header>
+    <div class="panel panel--unpadded">
+      <%= render "shared/profile-accordion" %>
+    </div>
+  </aside>
+
+  <aside class="with-left-sidebar__right">
+    <header class="panel-header">
+      <h2 class="panel-header__title">Assessments</h2>
+    </header>
+    <div class="panel panel--unpadded panel--stroked need--phone-triage">
+      <%= link_to "#{@type.humanize} an assessment", contact_path(@contact.id), class: "panel__header" %>
+      <div class="panel__body">
+        <% if @need.errors.any? || @note.errors.any? %>
+          <div class="error" id="error_explanation">
+            <h2><%= pluralize(@need.errors.count + @note.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+            <ul>
+              <% @need.errors.full_messages.each do |message| %>
+                <li><%= message %></li>
+              <% end %>
+              <% @note.errors.full_messages.each do |message| %>
+                <li><%= message %></li>
+              <% end %>
+            </ul>
+          </div>
+        <% end %>
+
+        <%= form_with(model: @need, url: contact_assessments_path, local: true) do |form| %>
+          <input type="hidden" value="<%= @type %>" name="type" hidden="hidden">
+          <div class="need-actions">
+            <div class="need-actions__field" id="assigned-actions__field">
+              <%= form.label :user_id, 'Assigned to', class: 'need-actions__label' %>
+              <%= form.collection_select :user_id, @users, :id, :name_or_email, {include_blank: "Unassigned"}, {class: "need-actions__select"} %>
+            </div>
+
+            <div class="need-actions__field" id="teams-actions__field">
+              <%= form.label :user_id, 'Team', class: 'need-actions__label' %>
+              <%= form.collection_select :role_id, @roles, :id, :name, {include_blank: "Unassigned"}, {class: "need-actions__select"} %>
+            </div>
+
+            <div class="need-actions__field" id="priority-actions__field">
+              <%= form.label :is_urgent, 'Priority', class: 'need-actions__label' %>
+              <%= form.select :is_urgent, {
+                  "Urgent" => true,
+                  "Normal" => false
+              }, {}, {class: @need.is_urgent ? "need-actions__select need-actions__select--red" : "need-actions__select"} %>
+            </div>
+
+            <div class="need-actions__field" id="status-actions__field">
+              <%= form.label :status, 'Status', class: 'need-actions__label' %>
+
+              <% if @type == 'log' %>
+                <%= form.select :status, [%w(Complete complete)], {}, {class: "need-actions__select", readonly: true} %>
+              <% else %>
+                <%= form.select :status, need_statuses, {}, {class: "need-actions__select"} %>
+              <% end %>
+            </div>
+          </div>
+
+          <div class="field">
```

Comment:
This should be wrapped in a `<fieldset>` and the label should be a `<legend>` (example in /views/contacts/_form_fields.erb.html) in order to use the app's radio styling. But feel free to leave it and create a card to fix that in the backlog, as it's just a minor styling issue.